Update README.md

jmaruland opened the pull request.

jmaruland committed:
Update README.md

danielballan commented on README.md:
```diff
@@ -1,6 +1,14 @@
 # AI Multimodal Public Demos
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
```
We should include all institutions involved in the grant.

dylanmcreynolds replied:
Should we include facilities as well as labs?

Labs: BNL, ANL, LBNL
Facilities: NSLS-II, APS, ALS, CNM, CFN

Right?

jmaruland replied:
@dylanmcreynolds  I can group the facilities by Lab. It would make it easier for the general public to understand how we are organized. Are these all of the facilities involved in the project?

danielballan commented on README.md:
```diff
@@ -1,6 +1,14 @@
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
+Current contributors working in this project are: Dylan McReynolds and Jospeh Kleinhenz from LBL, and Dan Allan and Juan Marulanda from BNL.
```
We should include all coauthors on the grant and all the post-docs hired through the grant.

jmaruland replied:
Do we have a list of all the post-docs in all the participating facilities?

danielballan commented on README.md:
```diff
@@ -1,6 +1,14 @@
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientist from [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/) and [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/)
+Current contributors working in this project are: Dylan McReynolds and Jospeh Kleinhenz from LBL, and Dan Allan and Juan Marulanda from BNL.
+Our goal is to create database for XAS data that can be used by scientist from beamlines at different Synchrtron Light Sources around the world. They will be able to use this database as a platform to share and use data to improve their research. 
```
I actually don't know if this is a goal.

danielballan replied:
This is definitely something that other related projects are doing, and it seems like something that we _could_ do, but it starts to sound like straying in Materials Project territory. I think the focus of this particular effort is to provide centralize access to _multimodal_ data sets from a variety of facilities in order to support multi-modal AI/ML analysis.

danielballan commented on README.md:
```diff
@@ -1,6 +1,14 @@
+Current contributors working in this project are: Dylan McReynolds and Jospeh Kleinhenz from LBL, and Dan Allan and Juan Marulanda from BNL.
+Our goal is to create database for XAS data that can be used by scientist from beamlines at different Synchrtron Light Sources around the world. They will be able to use this database as a platform to share and use data to improve their research. 
+
+We develop and used [tiled](https://blueskyproject.io/tiled/) for these demos. First, we can define Tiled as a data access service for data science tools and we use it to put together all the different data sets that we receive from beamline scientists and it helps users to avoid worrying about formatting, file structure and parsing.
```
I think we should say "integrate with" but not "develop" because core development of TIled is paid for by NSLS-II Operations, not AIMM.

dylanmcreynolds replied:
Agreed with @danielballan  We are building a funcitonal database of datasets from a variety of experiments and simulations. Oh, and we use Tiled.

jmaruland replied:
Shouldn't we mention here that we are also in charge of Tiled? We are developing Tiled for general use but it is inspired on on the goals of this project, right?

danielballan commented on README.md:
```diff
@@ -1,6 +1,14 @@
+
+We develop and used [tiled](https://blueskyproject.io/tiled/) for these demos. First, we can define Tiled as a data access service for data science tools and we use it to put together all the different data sets that we receive from beamline scientists and it helps users to avoid worrying about formatting, file structure and parsing.
+
+Currently, we have integrated data sets provided by beamline scientist from Argonne National Laboratory (ANL), Brookhaven National Laboratory (BNL) and Lawrence Berkeley National Laboratory (LBL). Some of these scientists are: Dr. Maria Chen (ANL), Dr. Wanli Yang (LBL), Dr. Chengjun Sun (ANL), Dr. Eli Stavitski (BNL), Dr. Steve Heald (ANL). We also received a letter of support from Dr. Matt Newvi ...
```
Can we break each of these items out into a bulleted list with a description of each database?

danielballan replied:
For now, let's keep it simple and list only the public data sets, which I guess only include Newville at the moment.

danielballan replied:
And we should link to the original source of the Newville data here too, to be friendly.

dylanmcreynolds commented on README.md:
```diff
@@ -1,6 +1,14 @@
+
+Currently, we have integrated data sets provided by beamline scientist from Argonne National Laboratory (ANL), Brookhaven National Laboratory (BNL) and Lawrence Berkeley National Laboratory (LBL). Some of these scientists are: Dr. Maria Chen (ANL), Dr. Wanli Yang (LBL), Dr. Chengjun Sun (ANL), Dr. Eli Stavitski (BNL), Dr. Steve Heald (ANL). We also received a letter of support from Dr. Matt Newvi ...
+
+The demos in this repostiory have access to both public and private data sets. You can run these sets by following the intructions below. Keep in mind that you will need authorization to access the private data sets. We will give authorization to a selective group of scientists via ORCID authetification.
```
Instead of 

> The demos in this repostiory have access to both public and private data sets

How about 

> This repository contains both public and private datasets.

dylanmcreynolds reviewed (changes requested):
I agree with most of Dan's comments. I want to emphasize that we want to be listing the data sets and what they contain, pending approval from the contributing scientists.

jmaruland committed:
Added changes from the first reviews from Dan and Dylan

More changes coming. Not all reviews were considered because there are some questions that need to be discussed first

jmaruland committed:
whitespace

danielballan commented on README.md:
```diff
@@ -1,6 +1,28 @@
+- [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/): Center for Functional Nanomaterials (CFN) and National Synchrotron Light Source II (NSLS-II)
+- [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/): Advanced Light Source (ALS)
+
+The current contributors and authors working in this project are:
```
@elistavitski Do you have handy a list of all the post-doc personnel on this project and their affiliations?

elistavitski replied:
ANL Postdocs: 
- Inhui Hwang
- Yiming Chen
BNL Scientists
- Matthew Carbone
BNL postdocs
- Zhu Liang 
- Fanchen Meng

elistavitski replied:
Is Pandolfi still on the project?

danielballan replied:
Ron Pandolfi asked to step back, if I understood his intentions correctly, so I think it would be appropriate to omit his name.

danielballan replied:
Also, I think we should remove the explicit "Scientists" and "Post-docs" designation. (Not all staff here are best classified as "scientists", like Dylan.) Just list everybody, with staff first.

jmaruland committed:
Added more changes after reading more reviews

jmaruland committed:
Added more names to project contributors

jmaruland committed:
whitespace

jmaruland committed:
whitespace

jmaruland committed:
whitespace

jmaruland committed:
Reformatted the list of contributors

danielballan reviewed (approved):
Looks good enough for a first pass to me. Can always continue to improve it as we go.

dylanmcreynolds commented on README.md:
```diff
@@ -1,12 +1,34 @@
+<!-- Our goal is to create database for XAS data that can be used by scientist from beamlines at different Synchrtron Light Sources around the world. They will be able to use this database as a platform to share and use data to improve their research. -->
+Our goal is to enable and accelerate scientific discovery by leveraging large, complex multimodal datasets generated across BES synchrotron facilities. We are developing shared, transferrable infrastructure to store, curate, analyze, interpret and disseminate the data.
+
+We integrated with [Tiled](https://blueskyproject.io/tiled/) for these demos. First, we can define Tiled as a data access service for data science tools and we use it to put together all the different data sets that we receive from beamline scientists and it helps users to avoid worrying about formatting, file structure and parsing.
```
I suggest deleting "First, we can define".

jmaruland committed:
Added latest changes to the README file

Includes final review from Dylan
